### PR TITLE
Npr history

### PR DESCRIPTION
**Pull request by chrishaid**
@almartin82 Yo! This is fun visualization:

image

Easily one of my favorites this year!  

I also cleaned up some over documented and undocumented arguments in other functions to drive our `R CMD check` warnings down.

### CODE REVIEW

**Comment by chrishaid** [comment]
ooh, did you notice that wercker builds have an estimated completion time that updates every second? Cool.

**Comment by chrishaid** [comment]
And man those builds are fast!

**Comment by almartin82** [comment]
I did not! That's crazy!

(mobile)

File: R/student_npr_history_plot.R [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+#'    SchoolName=="Mt. Bachelor Middle School",
+#'    TermName=="Spring 2013-2014") %>% select(StudentID) %>%
+#'    unique()
+#'
+#' student_npr_history_plot(
+#'     map_mv,
+#'     studentids = ids[1:80, "StudentID"],
+#'     measurementscale = "Reading")
+
+student_npr_history_plot <- function(mapvizieR_obj,
```
Comment (almartin82) [inline R/student_npr_history_plot.R:35]: i think this is one of the few places where we don't agree on linting.  what does lintr say -- your style, or mine?

**Comment by almartin82** [comment]
thanks for cleaning up those undocumented args.  this looks good to go.  we still need to have a lint-off and decide how to indent parameter args, but that's not a huge deal at all.

**Pull request merged**